### ISSUE TITLE
[Bug] Color-contrast issue flagged by service unexpectedly

### ISSUE THREAD

**Issue by lamaks** [microsoft/accessibility-insights-service#960]
The https://opensource.microsoft.com/ page scan returns WCAG 1.4.3 issues when run automated scan on a service. Validating the same page using AI Web and axe Chrome extensions will not flag this issue.

The service automation scan should be consistent with other AI tools.

The related issue https://github.com/dequelabs/axe-core/issues/1730

![image](https://user-images.githubusercontent.com/40576301/90188329-1c5aa680-dd70-11ea-9a3a-0db0f52710bd.png)
